### ISSUE TITLE
Latest Release Error verifying GPG signature for provider "tfe"

### ISSUE THREAD

**Issue by caiodelgadonew** [hashicorp/terraform-provider-tfe#298]
The provider seems to be unsigned, not sure if something has been broken on the CI/CD steps but we're unable to use it.

This occurs on the release 0.25 that's the latest launched today.

Every apply throws that error: 

```
Error verifying GPG signature for provider "tfe"
Terraform was unable to verify the GPG signature of the downloaded provider
files using the keys downloaded from the Terraform Registry. This may mean that
the publisher of the provider removed the key it was signed with, or that the
distributed files were changed after this version was released.
```

**Comment by caiodelgadonew** [comment]
@chrisarcand do you know what is needed to fix it?

**Comment by chrisarcand** [comment]
👋  We'll look in to it immediately, thanks for reporting!

**Comment by chrisarcand** [comment]
@caiodelgadonew Ok, gotten to the bottom of it, I believe!

TLDR: You'll need to upgrade Terraform to the latest patch version to use a new signing key (new releases were made for every 'major' (minor) version going back to 0.11)

We (HashiCorp) have recently rotated our release signing key which verifies provider binaries during `terraform init`; you can read all about the potential effects that has for Terraform specifically here: [Terraform Updates for HCSEC-2021-12](https://discuss.hashicorp.com/t/terraform-updates-for-hcsec-2021-12/23570)

Of most importance to you here is likely this:

![image](https://user-images.githubusercontent.com/2430490/116554631-09b5dc80-a8c1-11eb-9750-d0c5a84dd466.png)

* I would guess you're probably using Terraform 0.11 or 0.12? If so, you'll need to upgrade to the latest patch version which includes the new key for verification and will fix that error.
* If you're using Terraform 0.12 and _unable_ to upgrade to the latest patch version for some reason, you can pin (add a constraint) this provider to use v0.24.0 until you're able to do so; info on that here: https://www.terraform.io/docs/language/providers/requirements.html. Constraining the provider is generally a good idea anyway; see more here: https://registry.terraform.io/providers/hashicorp/tfe/latest/docs#versions
* Random aside: If you're using Terraform 0.11, note that support for that version of Terraform with this provider ended in the previous release!

**Comment by caiodelgadonew** [comment]
Thanks @chrisarcand ! 

I'll do as you said there! Seems that this is the solution.

Just for clarifications, we're using 0.12.29 and 0.13.5.

So Updating that should work, thanks!

**Comment by chrisarcand** [comment]
(For posterity: We posted a pinned issue to raise awareness of a change we made to the Registry which now emits a much more helpful warning: #301)  🙇
